Files/UserControls/SidebarControl.xaml.cs

```diff
@@ -229,6 +229,7 @@ private void NavigationViewLocationItem_RightTapped(object sender, RightTappedRo
             // Additional check needed because ShowProperties is set to true if not recycle bin
             if (item.IsDefaultLocation)
             {
+                ShowEjectDevice = false;
```

Comment:
Should we put this outside the "if"?

Reply:
It shouldn't make a difference, the main thing is that the eject option shouldn't show for a regular pinned item.

Reply:
You sure? Because user pinned folders IsDefaultLocation is false I think